tools/integration/test/testConfig.js

```diff
@@ -25,12 +27,12 @@ const components = [
   'gem/rubygems/-/sorbet/0.5.11226',
   'pypi/pypi/-/platformdirs/4.2.0',
   'go/golang/rsc.io/quote/v1.3.0',
-  'nuget/nuget/-/NuGet.Protocol/6.7.1'
-  // 'composer/packagist/symfony/polyfill-mbstring/1.11.0',
-  // 'pod/cocoapods/-/SoftButton/0.1.0',
-  // 'deb/debian/-/mini-httpd/1.30-0.2_arm64'
-  // 'debsrc/debian/-/mini-httpd/1.30-0.2_arm64',
-  // 'sourcearchive/mavencentral/org.apache.httpcomponents/httpcore/4.1'
+  'nuget/nuget/-/NuGet.Protocol/6.7.1',
+  'composer/packagist/symfony/polyfill-mbstring/v1.28.0',
+  // 'pod/cocoapods/-/SoftButton/0.1.0', // Dev and prod have different file counts. See https://github.com/clearlydefined/crawler/issues/529
```

Comment:
Dev crawler is node 18, and prod crawler is node 16.  The git version in node 16 and node 18 are different, causing [this issue](https://github.com/clearlydefined/crawler/issues/529).  Would really appreciate some feedback on the [proposed fix](https://github.com/clearlydefined/crawler/pull/525)